src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/controllers/applications/respondent/TseRespondentReplyController.java

```diff
@@ -115,7 +115,7 @@ public ResponseEntity<CCDCallbackResponse> showError(
         CaseData caseData = ccdRequest.getCaseDetails().getCaseData();
         List<String> errors = new ArrayList<>();
         if (Helper.isClaimantNonSystemUser(caseData)
-                && Boolean.FALSE.equals(Helper.isRepresentedClaimantWithMyHmctsCase(caseData))) {
+            && !Helper.isRepresentedClaimantWithMyHmctsCase(caseData)) {
```

Comment:
Same as the comment above.

Reply:
I have not refactored as this logic is checking a true and a false and inverting it would still result in the same check. Further code change is required which will cause changes to code outside of applications which is not the intention of this PR